CONTRIBUTING.rst

```diff
@@ -1,56 +1,55 @@
-# James Webb Space Telescope Data Analysis Tool Notebooks
+============================
+Contribute to JDAT Notebooks
+============================
 
-   ``jdat_notebooks`` are one component of STScI's larger [Data Analysis Tools Ecosystem](https://jwst-docs.stsci.edu/jwst-post-pipeline-data-analysis).
-
-   The [jdat_notebooks repository](https://github.com/spacetelescope/jdat_notebooks) is for already-reviewed notebooks that are ready for consumption by the science community.  If you wish to contribute notebooks or technical expertise, we refer you to [dat_pyinthesky](https://github.com/spacetelescope/dat_pyinthesky/tree/master/jdat_notebooks) as the place to submit a first draft.  The description below outlines the whole process, which begins in `dat_pyinthesky` but finishes in `jdat_notebooks`.
-
-
-# Development Procedure for JDAT Notebooks
-
-This document is a description of the JWST Data Analysis Tools (JDAT) Approach to
-"Notebook-Driven Development".  The procedures here outline the process for
+This document outlines the process for contributing code to the jdat notebooks by following the JWST Data Analysis Tools (JDAT) Approach to
+"Notebook-Driven Development". The procedures here outline the process for
 getting a notebook through successive development stages to become something
 that can be "live" on the spacetelescope notebooks repository.
 
 These notebooks can have many varied science cases, but follow a relatively
-standard workflow:
+standard workflow
 
-1. Draft Stage
-2. Baseline Stage
-3. Notebook-driven development
-4. Advanced Stage
-5. Final Notebook - Maintenance Stage
-6. Revision based on Community feedback
+1. `Draft Stage`_
+2. `Baseline Stage`_
+3. `Notebook-driven development`_
+4. `Advanced Stage`_
+5. `Final Notebook - Maintenance Stage`_
+6. `Revision based on Community feedback`_
 
 These stages and the process for moving from one to the other are described below.
 
 **The procedure to submit the notebook via a Pull Request is described at the end of this document.
 This is repeated for each of the 5 stages.**
 
 Note that there is much more information on writing Jupyter notebooks at the
-[STScI notebook style guide](https://github.com/spacetelescope/style-guides/blob/master/guides/jupyter-notebooks.md),
+`STScI notebook style guide <https://github.com/spacetelescope/style-guides/blob/master/guides/jupyter-notebooks.md>`_,
 and similar guidance for Python code at the
-[STScI Python style guide](https://github.com/spacetelescope/style-guides/blob/master/guides/python.md).
+`STScI Python style guide <https://github.com/spacetelescope/style-guides/blob/master/guides/python.md>`_.
 These guidelines are in place to make review steps easier.
 
-## Draft Stage
+.. important::
```

Comment:
This renders funny. I think it is fine to just add the following sentence to the previous sentence, deleting this "Important".

Reply:
Initially, I considered taking a similar approach to dat_pyinthesky regarding its appearance. However, if that's the preferred option, I am perfectly content with a more straightforward text-based approach.

<img width="690" alt="image" src="https://user-images.githubusercontent.com/66814693/220734240-915e7912-71cb-443a-a3c6-3d09bd429041.png">

Reply:
I see. If you can find a way to render it like in your screenshot, that is nice. The way it is now, (or at least they way I see it on the github page) looks weird.
<img width="851" alt="screenshot" src="https://user-images.githubusercontent.com/19998720/220745242-7ebbf6a8-f8e6-4f5a-9d7b-825737e602eb.png">